lib/type/i-flow-purchase-event.ts

```diff
@@ -0,0 +1,5 @@
+export interface IFlowPurchaseEvent {
```

Comment:
This one is app-specific and shouldn't be included

Reply:
Removed